app/main.py

```diff
@@ -1 +1,7 @@
-# write your code here
+def copy_file(command: str) -> None:
+    file1 = command.split()[1]
+    file2 = command.split()[2]
```

Comment:
first you should check that the length of `command.split()` is not less than three